symfony/bluesky-notifier/7.1/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "#1": "BLUESKY_DSN=bluesky://USER:PASS@bsky.social"
```

Comment:
```suggestion
        "#1": "BLUESKY_DSN=bluesky://USER:PASS@default"
```
?

Reply:
AFAIU, there is no default.

Reply:
You mean in our bridge or in general for Bluesky? I opened a PR
* https://github.com/symfony/symfony/pull/53094